src/amqproxy/pool.cr

```diff
@@ -2,7 +2,7 @@ module AMQProxy
   class Pool
     getter :size
 
-    def initialize(@host : String, @port : Int32, @tls : Bool, @log : Logger, @idle_connection_timeout : Int32)
+    def initialize(@host : String, @port : Int32, @tls : Bool, @metrics_client : MetricsClient, @log : Logger, @idle_connection_timeout : Int32)
```

Comment:
make it the last parameter and give it a default value

Reply:
Updated in https://github.com/cloudamqp/amqproxy/pull/68/commits/8be641fb28a3792b46fd66d88b24be0667454e4e